x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/utils/ToXContentParams.java

```diff
@@ -14,16 +14,10 @@
     /**
      * Parameter to indicate whether we are serialising to X Content for
      * internal storage. Certain fields need to be persisted but should
-     * not be visible everywhere.
+     * not be visible everywhere
```

Comment:
Why remove the full stop at the end of the sentence?

Reply:
Added it back :)